src/features/reports/pages/reports/DeliveryDetailView.tsx

```
@@ -0,0 +1,119 @@
+  const { delivery, loading, error } = useReportDeliveryDetail(practiceId, deliveryId);
+  const { navigate } = useNavigation();
+
+  const backToList = () => {
+    if (!practiceSlug) return;
+    navigate(`/practice/${encodeURIComponent(practiceSlug)}/reports/deliveries`);
+  };
+
+  if (loading) {
+    return (
+      <div className="flex min-h-0 flex-1 flex-col gap-4 p-4 sm:p-6">
+        <p className="text-sm text-input-placeholder">Loading delivery…</p>
```

ESLint

[error] Visible loading text is not allowed. Use LoadingSpinner, LoadingBlock, LoadingScreen, or SkeletonLoader instead.

(custom/loading-consistency)